ur5_moveit_config/launch/moveit_rviz.launch

```diff
@@ -11,6 +12,7 @@
   <node name="$(anon rviz)" launch-prefix="$(arg launch_prefix)" pkg="rviz" type="rviz" respawn="false"
 	args="$(arg command_args)" output="screen">
     <rosparam command="load" file="$(find ur5_moveit_config)/config/kinematics.yaml"/>
+    <param name="arm_prefix" value="$(arg arm_prefix)"/>
```

Comment:
The URKinematics plugin is loaded into RViz when the kinematics.yaml is parsed. The kinematics.yaml tells RViz what plugin to use. Since RViz is spawned in this stanza of the launch file, it is the "parent". The URKinematics plugin is thus equal to the namespace of RViz, and will look for ~params (Private params) with respect to its namespace (in this case it is RVIz because it is a plugin).

So the param, if put anywhere else but the RViz namespace, won't be readable by the URKinematics plugin (because of how it is written in code).

See 

     private_handle.param<std::string>("arm_prefix", arm_prefix_, "");

in https://github.com/ros-industrial/universal_robot/blob/hydro-devel/ur_kinematics/src/ur_moveit_plugin.cpp